bindings/kvp/src/main/java/org/n52/sos/decode/kvp/AbstractKvpDecoder.java

```diff
@@ -337,13 +339,13 @@ protected Time parseTime(String parameterValue, String parameterName) throws Cod
         return namespaces;
     }
 
+
     private TemporalFilter createTemporalFilterFromValue(String value, String valueReference)
             throws OwsExceptionReport, DateTimeParseException {
         TemporalFilter temporalFilter = new TemporalFilter();
         temporalFilter.setValueReference(valueReference);
         String[] times = value.split("/");
-
-        if (times.length == 1) {
+        if (times.length == 1 && !temporalFilter.getOperator().equals(TimeOperator.TM_During)) {
```

Comment:
Ooh, good catch. This is not supposed to be here. I was working on develop first and when I moved back to feature/AI this somehow stayed in.